scripts/update-network-names-doc.ts

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs')
+const path = require('path')
+const { networks } = require("@0xsequence/network/constants");
+
+const networkPath = path.resolve(__dirname, '../packages/network')
+const templatePath = path.join(networkPath, 'networkNames.template.md')
+const outputPath = path.join(networkPath, 'networkNames.md')
+const template = fs.readFileSync(templatePath, 'utf8')
+
+const networkObjs = Object.values(networks) as Array<{
+  chainId:number, 
+  name:string, 
+  title?:string
+}>;
+
+networkObjs.sort((a, b) => a.chainId - b.chainId);
+const lines:string[] = [];
+lines.push(`| Chain ID | Name Slug |`);
+lines.push(`| --- | --- |`);
+for (const n of networkObjs) {
+  lines.push(`| ${n.chainId} | ${n.name} |`);
+}
+
+lines.push(""); 
+
+networkObjs.sort((a, b) => a.title?.localeCompare(b.title || "") || 0);
+lines.push(`| Chain Name | Name Slug |`);
+lines.push(`| --- | --- |`);
+for (const n of networkObjs) {
+  lines.push(`| ${n.title} | ${n.name} |`);
+}
```

Comment:
Can we combine `id | slug | name` into a single table?

Reply:
Yeah, I was waffling about that because chain ids and alphabetical name are different orders, and I wasn't sure which one users would prefer, so I put both.
But that said, users can use CMD+F honestly.
How about 
`Name | ID | slug`
So that the name, alphabetically, will be the primary sort

Reply:
I'll leave order up to you, whatever makes more sense from user perspective :)

Reply:
Let's combine into a single table please @bunnybones1 it will be easier to parse